src/compile/kinds/native.ts

```diff
@@ -0,0 +1,158 @@
+import axios from "axios";
+import { spawn } from "child_process";
+import fse from "fs-extra";
+import os from "os";
+import path from "path";
+import * as stream from "stream";
+import { promisify } from "util";
+import { SolcInput } from "../input";
+import { Compiler } from "./compiler";
+
+export function getCompilerPrefixForOs(): string | undefined {
+    const arch = os.arch();
+
+    /**
+     * Only 64 bit native compilers built
+     */
+    if (arch !== "x64" && arch !== "ia64") {
```

Comment:
Actually, this should also only be "x64". "ia64" is a different architecture, and solidity is not compiled for it.

Reply:
Fixed in https://github.com/ConsenSys/solc-typed-ast/pull/90.